tests/test_types_payment_card_number.py

```diff
@@ -0,0 +1,81 @@
+from collections import namedtuple
+from typing import Any
+
+import pytest
+
+from pydantic import BaseModel, ValidationError
+from pydantic.types import PaymentCardBrand, PaymentCardNumber
+
+VALID_AMEX = '370000000000002'
+VALID_MC = '5100000000000003'
+VALID_VISA = '4000000000000002'
+VALID_OTHER = '2000000000000000008'
+LUHN_INVALID = '4000000000000000'
+LEN_INVALID = '40000000000000006'
+
+
+class PaymentCard(BaseModel):
+    card_number: PaymentCardNumber
+
+
+def test_validate_digits():
+    digits = '12345'
+    assert PaymentCardNumber.validate_digits(digits) == digits
+    with pytest.raises(ValueError):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match=...):
```
to be a bit more exact.